cms/static/cms/js/modules/cms.modal.js

```diff
@@ -557,19 +591,38 @@
                 return url;
             },
 
-            _loadContent: function (url, name) {
+            /**
+             * load the iframe
+             *
+             * @method _loadIframe
+             * @param opts
+             * @param opts.breadcrumb {Array} collection of breadcrumb items
```

Comment:
what's the signature of breadcrumb item? it may be useful to make this `{Object[]}` or `{String[]}` instead